### PR TITLE
Fix some flaky tests

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

Fix some flaky tests

## Fixes

https://github.com/valory-xyz/open-aea/issues/517

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

DELETE INCLUSIVE THIS AND BELOW FOR STANDARD PR
------

## Release summary

Version number: [e.g. 1.0.1]

## Release details

Describe in short the main changes with the new release.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] Lint and unit tests pass locally and in CI
- [ ] I have checked the fingerprint hashes are correct by running (`aea packages lock --check`)
- [ ] I have regenerated the latest API docs
- [ ] I built the documentation and updated it with the latest changes
- [ ] I have added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

File: plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py [hunk 1]
```diff
@@ -63,14 +64,23 @@
...
+
+    # do modules cleanup to have fresh on import
+    for module_name in list(sys.modules.keys()):
+        for prefix in ["cosmpy.", "cosmos."]:
+            if module_name.startswith(prefix):
+                sys.modules.pop(module_name)
+
+    from cosmpy.protos.cosmos.base.v1beta1.coin_pb2 import (  # isort: skip  # Load Coin first
+        Coin,
+    )
```
Comment (angrybayblade) [inline plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py:76]: Please add a comment on what test this fixes

**Comment by angrybayblade** [comment]
Can you take a look into this failure https://github.com/valory-xyz/open-aea/actions/runs/4127177004/jobs/7130149417#step:6:1222

**Comment by solarw** [comment]
> Can you take a look into this failure https://github.com/valory-xyz/open-aea/actions/runs/4127177004/jobs/7130149417#step:6:1222

looks like network issue. test already wrapped with rerun

**Review by angrybayblade — DISMISSED** [review]
LGTM, but this test has failed several times https://github.com/valory-xyz/open-aea/actions/runs/4127177004/jobs/7140017960#step:6:1109 please take a look

**Comment by solarw** [comment]
> LGTM, but this test has failed several times https://github.com/valory-xyz/open-aea/actions/runs/4127177004/jobs/7140017960#step:6:1109 please take a look


remote service issue
```
Error: 2-09 07:46:48,226][ERROR] error on download of bafybeidjvb4ojaw2trxu4rlxq3blppfherkldwz4x5spnpvef5n34jvmmm: HTTPError: 504 Server Error: Gateway Time-out for url: [https://registry.autonolas.tech:443/api/v0/get?stream-channels=true&archive=true&compress=false&arg=bafybeidjvb4ojaw2trxu4rlxq3blppfherkldwz4x5spnpvef5n34jvmmm](https://registry.autonolas.tech/api/v0/get?stream-channels=true&archive=true&compress=false&arg=bafybeidjvb4ojaw2trxu4rlxq3blppfherkldwz4x5spnpvef5n34jvmmm)
[2023-02-09 07:46:49,230][DEBUG] Starting new HTTPS connection (1): registry.autonolas.tech:443
[2023-02-09 07:47:49,576][DEBUG] [https://registry.autonolas.tech:443](https://registry.autonolas.tech/) "POST /api/v0/get?stream-channels=true&archive=true&compress=false&arg=bafybeidjvb4ojaw2trxu4rlxq3blppfherkldwz4x5spnpvef5n34jvmmm HTTP/1.1" 504 160
Error: 2-09 07:47:49,578][ERROR] error on download of bafybeidjvb4ojaw2trxu4rlxq3blppfherkldwz4x5spnpvef5n34jvmmm: HTTPError: 504 Server Error: Gateway Time-out for url: [https://registry.autonolas.tech:443/api/v0/get?stream-channels=true&archive=true&compress=false&arg=bafybeidjvb4ojaw2trxu4rlxq3blppfherkldwz4x5spnpvef5n34jvmmm](https://registry.autonolas.tech/api/v0/get?stream-channels=true&archive=true&compress=false&arg=bafybeidjvb4ojaw2trxu4rlxq3blppfherkldwz4x5spnpvef5n34jvmmm)
```

**Comment by solarw** [comment]
> LGTM, but this test has failed several times https://github.com/valory-xyz/open-aea/actions/runs/4127177004/jobs/7140017960#step:6:1109 please take a look

updated

**Review by Adamantios — APPROVED** [review]
PR is about adding reruns for flaky tests. Looks good if @angrybayblade is fine with this and we do not want to prioritize finding out the cause right now.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**